Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.19 as builder
+FROM golang:1.20 as builder
```

Comment:
if we do that here, i assume we should do it on other places in the project too. :x: